src/pageEditor/tabState/tabStateSlice.ts

```
@@ -133,6 +134,7 @@ export const tabStateSlice = createSlice({
           "PixieBrix was updated or restarted. Reload the Page Editor to continue."
         );
         state.error = serializeError(error);
+        reportError(error);
```

This should be dropped, context invalidation errors are not reported.